api/external/exports.html

```diff
@@ -232,9 +232,9 @@ <h1>Exports<a class="headerlink" href="#exports" title="Permalink to this headli
 <pre class="response">
 <b>GET /100/exports/200/download</b>
 
-email,member_id,member_since,plaintext_preferred,bounce_count,status_name,last_modified_at,status-name,last_name,custom-field-name
-email@email.com,123456789,2016-10-28T15:05:52 UTC,f,0,active,2019-05-07T20:44:21 UTC,,Last_name,,,2.0,37210,,,,b,,,,,,,,,,,,,
-email1@email.com,234567890,2019-05-02T17:47:11 UTC,f,0,active,2019-05-02T17:47:11 UTC,,Member,,,,,,,,,
+email,member_id,member_since,plaintext_preferred,bounce_count,status_name,last_modified_at,first_name,last_name,sms-test,sms_phone_number,promotional_status,transactional_status,region_code
+email@email.com,123456789,2022-09-16T20:39:39 UTC,f,0,active,,first_name,last_name,,,,,
+email@email.com,123456789,2022-09-19T15:04:01 UTC,f,0,active,2022-09-19T15:22:14 UTC,,,,+16155555555,a,a,
```

Comment:
Looks like `sms-test` just happens to be a custom field in the account we were testing with, and I'm a li'l concerned it might be confusing:

```suggestion
email,member_id,member_since,plaintext_preferred,bounce_count,status_name,last_modified_at,first_name,last_name,sms_phone_number,promotional_status,transactional_status,region_code
email@email.com,123456789,2022-09-16T20:39:39 UTC,f,0,active,,first_name,last_name,,,,
email@email.com,123456789,2022-09-19T15:04:01 UTC,f,0,active,2022-09-19T15:22:14 UTC,,,+16155555555,a,a,
```